### DISCUSSION TITLE
add migration support from other knowledge bases

### DISCUSSION THREAD

**Discussion by ReddixT** [lumium-technologies/lumium#199] [Ideas]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
Ability to export and import to from another kb

**Describe alternatives you've considered**

**Additional context**
a feature for the future

**Comment by cedricschwyter** [comment]
which knowledge bases do you want to support and how do you want to do this? afterall, at the moment we will only support markdown files, so we need to somehow either directly import markdown files or have an individual converter for a specific knowledge base

**Reply by ReddixT** [reply]
i mean for specific programs we need to find out how they export it so well see. it will be an idea in the future

**Reply by cedricschwyter** [reply]
sure, but we can already start to compile a list of platforms we want to support.
